addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/TestPathTraversalUnitTest.java

```diff
@@ -51,7 +53,7 @@ protected int getRecommendMaxNumberMessagesPerParam(AttackStrength strength) {
             case HIGH:
                 return recommendMax + 7;
             case INSANE:
-                return recommendMax;
+                return 300;
```

Comment:
hmm. I was also thinking the same.
till now i have added following extension -
 ".html", ".asp", ".php", ".jsp", ".jspf", ".txt", ".htm", ".aspx", ".config"

I think config, txt, htm and jspf  can be removed.

Reply:
Why those extensions and not others? IMO it would be better to use the parameter value (which is most likely something that's valid/accepted by the application) than add an arbitrary extension.

Reply:
hmm. this is a good idea. will do the same.

Reply:
Hi @thc202 
I have updated the code as you suggested.
thanks,
Karan